reference/library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -34,8 +34,7 @@ $header-font-weight: 700 !default;										// 400 or 700 as you wish
 $background-color: 			 #FFFFFF !default;
 $background-color-secondary: #444444 !default;
 
-$primary-color:    			 #2a94c0 !default;
-$primary-color-shadow:			 #2A87C0 !default;
```

Comment:
The primary-color-shadow was added in #2896 which looks like it involves improving the look of the minimized flags. I realize there's confusion around these variables and maybe it could use a sass script rather than a fixed color and still be included? It looked like the old one was hue +5 so

$primary-color-shadow:adjust-hue($primary-color,5) !default;